pkg/exec/exec.go

```diff
@@ -37,70 +36,85 @@ var (
 	CephClusterNamespace string // Cephcluster namespace
 )
 
-func RunCommandInOperatorPod(ctx context.Context, clientsets *k8sutil.Clientsets, cmd string, args []string, operatorNamespace, clusterNamespace string, returnOutput, exitOnError bool) string {
+func RunCommandInOperatorPod(ctx context.Context, clientsets *k8sutil.Clientsets, cmd string, args []string, operatorNamespace, clusterNamespace string, returnOutput bool) (string, error) {
 	var pod v1.Pod
 	var err error
 
 	pod, err = k8sutil.WaitForPodToRun(ctx, clientsets.Kube, operatorNamespace, "app=rook-ceph-operator")
 	if err != nil {
-		logging.Fatal(fmt.Errorf("failed to wait for operator pod to run: %v", err))
+		return "", fmt.Errorf("failed to wait for operator pod to run. %w", err)
 	}
 
 	var stdout, stderr bytes.Buffer
 
-	execCmdInPod(ctx, clientsets, cmd, pod.Name, "rook-ceph-operator", pod.Namespace, clusterNamespace, args, &stdout, &stderr, returnOutput, exitOnError)
+	err = execCmdInPod(ctx, clientsets, cmd, pod.Name, "rook-ceph-operator", pod.Namespace, clusterNamespace, args, &stdout, &stderr, returnOutput)
 	if !returnOutput {
-		return ""
+
+		return "", err
 	}
 
 	fmt.Print(stderr.String())
```

Comment:
dont we need to consider `stderr` for anything? currently its treated with no value except for printing

Reply:
Wrapped `err` by `stderr` string. So caller can use it.